packages/core/useVirtualList/index.ts

```diff
@@ -155,7 +158,7 @@ function createGetOffset<T>(source: UseVirtualListResources<T>['source'], itemSi
         break
       }
     }
-    return offset + 1
+    return offset
```

Comment:
This sounded like an off by 1. Without this, change, we can never have an offset of 0. With 0 overscan, that would mean we can never render the very start of the list.